frontend/client/__tests__/addmembermodal.react.test.js

```diff
@@ -0,0 +1,153 @@
+import { JSDOM } from 'jsdom'
+
+const dom = new JSDOM('<!doctype html><html><body></body></html>')
+global.window = dom
+global.document = dom.document
```

Comment:
I think we can delete everything above this line

Reply:
See Manage Members test PR